_events/2018-06-27-de.STAIR-primer-rna-seq.md

```diff
@@ -0,0 +1,24 @@
+---
+site: Jena
+tags:
+title: A Primer for RNA-Sequencing Processing, Interpreting and Visualization 27.-29.06.2018 in Jena
+starts: 2018-06-27
+ends: 2018-06-29
+organiser:
+  name: de.STAIR
+  email: destair@leibniz-fli.de
```

Comment:
isn't this destairdenbi@gmail.com ?

Reply:
@bgruening: correct, but we're using @leibniz-fli.de for organising events and replying to participants' questions after the event is over